templates/accounts/multi_email_cleanup.html

```diff
@@ -0,0 +1,28 @@
+{% extends "accounts/_section.html" %}
+
+
+{% block title %}edit account{% endblock %}
+
+{% block section_content %}
+
+    <h1>Your email address should be updated</h1>
+
+    <p>Dear <b>{{ request.user.username }}</b>, we recently introduced a new pocily in Freesound which does not allow
+        different user accounts to share a single email address. Only a few users are affected by this change, and you
+        happen to be one of them. But don't worry, the fix is easy ;)</p>
+
+    <p>All what you have to do is to either set a new email using the <a href="{% url 'accounts-email-reset' %}">reset
+        email form</a>, or submit a <a href="{% url 'contact' %}">support request</a> telling us your username
+        and the new email you want to use for your account.</p>
+
+    <p>FYI, your original email address was <b>{{ original_email }}</b>, and was used by the following accounts:</p>
```

Comment:
For your information, the email address which your accounts are using is ..., and is used by the following accounts